packages/renderer/src/lib/kube/resource-listen.spec.ts

```diff
@@ -0,0 +1,341 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { CoreV1Event, KubernetesObject } from '@kubernetes/client-node';
+import { type Writable, writable } from 'svelte/store';
+import { afterEach, beforeEach, describe, expect, test, vi } from 'vitest';
+
+import type { IDisposable } from '../../../../main/src/plugin/types/disposable';
+import { listenResource } from './resource-listen';
+import * as resourcesListen from './resources-listen';
+
+vi.mock(import('./resources-listen'), async importOriginal => {
+  const original = await importOriginal();
+  return {
+    // we want to access the original isKubernetesExperimentalMode
```

Comment:
question: why? why not just mocking the `isKubernetesExperimentalMode`?

Reply:
Yes, you are right, I'll change to simplify